examples/http/nginx_status.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * this driver is based on nginx module "ngx_http_stub_status_module"
+ * You need to read this http://nginx.org/en/docs/http/ngx_http_stub_status_module.html 
+ * before using this driver
+ */
+// nginx status page config
+var nginxStatusHttpConfig = {
+    port: 10011,
+    url: "/nginx_status"
```

Comment:
Maybe its best to use the example link one (basic_status):

<img width="643" alt="image" src="https://user-images.githubusercontent.com/3306549/194096064-8cc14a10-f35f-493a-a37e-b10564c32211.png">